apps/reflect/src/Context/ColorModeContext.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactNode, createContext, useEffect, useState } from "react";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import { getDesignTokens, getThemedComponents } from "src/utils/theme";
+
+interface ColorModeContextType {
+  toggleColorMode: () => void;
+  colorMode: "dark" | "light";
+}
+
+export const ColorModeContext = createContext<ColorModeContextType>({
+  toggleColorMode: () => {},
+  colorMode: "dark",
+});
+
+export const ColorModeProvider = ({ children }: { children: ReactNode }) => {
```

Comment:
Using 'FC' from 'import type {FC} from 'react'' to define types is important for the integrity of the code in the repository